packages/blocks/src/compositionObjects.ts

```diff
@@ -29,27 +25,25 @@ export const PlainText = (text: string, emoji = true): PlainTextElement => ({
 // --- Confirm Object --- https://api.slack.com/reference/block-kit/composition-objects#confirm
 
 // --- Option Object --- https://api.slack.com/reference/block-kit/composition-objects#option
-const optionTruncates: TruncateOptions = {
-  text: [75, ellipsis],
-  value: [75, disallow],
-  description: [75, ellipsis],
-  url: [3000, truncate],
-};
-
 export const OptionObject = (
   text: string,
   value: string,
   optionBlock: Partial<Option> = {},
-  truncateFunctions: Record<string, TruncateFunction> = {},
+  overrideTruncators: Record<string, TruncateFunction> = {},
 ): Option =>
-  applyTruncations<Option>(
+  applyTruncationsWithOverrides<Option>(
     {
       text: PlainText(text),
       value,
       ...optionBlock,
-    } as Option,
-    mergeLeft(truncateFunctions, truncators(optionTruncates)),
-    truncLimits(optionTruncates),
+    },
+    {
+      text: [75, ellipsis],
+      value: [75, disallow],
+      description: [75, ellipsis],
+      url: [3000, truncate],
+    },
+    overrideTruncators,
```

Comment:
updated the way we call this so that's a bit less duplicative and easier to build for multiple elements - some of this logic was moved into the `applyTruncationsWithOverrides` lengthHelper. I think it more aptly communicates what's going on, too.